AzureSpatialAnchorsSample/Packages/packages-lock.json

```diff
@@ -1,7 +1,7 @@
 {
   "dependencies": {
     "com.microsoft.azure.spatial-anchors-sdk.core": {
-      "version": "file:../../MixedRealityPackages/com.microsoft.azure.spatial-anchors-sdk.core-2.10.0-preview.1.tgz",
+      "version": "file:../../MixedRealityPackages/com.microsoft.azure.spatial-anchors-sdk.core-2.10.1.tgz",
```

Comment:
the title of the PR said ".2" but here the code is ".1".  need to fix one of them.

Reply:
It should say .2 in the code as well now; I think you started reviewing in the middle of me pushing the second update.